packages/react/src/runtime/__perf__/ax.test.ts

```diff
@@ -3,24 +3,24 @@ import { runBenchmark } from '@compiled/benchmark';
 import { ax } from '../index';
 
 describe('ax benchmark', () => {
-  it('completes with ax() string as the fastest', async () => {
-    const arr = [
-      '_19itglyw',
-      '_2rko1l7b',
-      '_ca0qftgi',
-      '_u5f319bv',
-      '_n3tdftgi',
-      '_19bv19bv',
-      '_bfhk1mzw',
-      '_syazu67f',
-      '_k48p1nn1',
-      '_ect41kw7',
-      '_1wybdlk8',
-      '_irr3mlcl',
-      '_1di6vctu',
-      undefined,
-    ];
+  const arr = [
+    '_19itglyw',
+    '_2rko1l7b',
+    '_ca0qftgi',
+    '_u5f319bv',
+    '_n3tdftgi',
+    '_19bv19bv',
+    '_bfhk1mzw',
+    '_syazu67f',
+    '_k48p1nn1',
+    '_ect41kw7',
+    '_1wybdlk8',
+    '_irr3mlcl',
+    '_1di6vctu',
+    undefined,
```

Comment:
Why is there `undefined` in here?

Reply:
`undefined` is an acceptable parameter so we want to include it in the test case.

```
ax([undefined, '.aaaabbbb'])
```

btw, I didn't add `undefined`.

Reply:
Optional: I know the `undefined` wasn't added by your PR, but I think incorporating what you've said here (`undefined` is a valid parameter to `ax`) into a brief comment in the code itself might be helpful for future developers. I know that I'd definitely be confused to see it in a test case as well 😅

Reply:
- [ ] Add comment